extensions/asde-deployment/package.json

```diff
@@ -26,7 +26,7 @@
         "platforms": "*",
         "icon": {
           "light": "./images/sqldb_edge.svg",
-          "dark": "./images/sqldb_edge_inverse.svg"
```

Comment:
I think icon property can also be a string, this can be simplified to "icon": "./images/sqldb_edge.svg", also apply this to these other 2 icons.

Reply:
We didn't but I fixed that